### PR TITLE
feat(send): atlassian_mu services in Python

### PR DESCRIPTION
**Pull request by Kuliak**
* atlassian_mu script rewritten into Python using send_lib library
* atlassian_mu_process.pl file is not part of this change

### CODE REVIEW

File: send/atlassian_mu [hunk 1]
```diff
@@ -1,109 +1,45 @@
...
-			trap 'rm -r -f "${LOCK_FILE}"' EXIT
-			echo $$ > "$LOCK_PIDFILE"
-			if [ $? -ne 0 ]; then
-				echo "Can't create lock file after removing invalid lock file." >&2
-				exit 246
-			fi
-		fi
-	fi
-}
+def check_input_args():
```
Comment (xflord) [inline send/atlassian_mu:15]: please use send_lib.check_input_fields() along with check_destination_type_allowed() and check_destination_format()

File: send/atlassian_mu [hunk 2]
```diff
@@ -1,109 +1,45 @@
...
-ERRORCODE=$?
-if [ $ERRORCODE -ne 0 ]; then
-	echo "Process exit with error" >&2
-fi
+	# create lock to ensure only one script of this type is running
+	send_lib.create_lock(SERVICE_NAME, destination)
 
-exit $ERRORCODE
+	# prepare temporary working directory and copy all needed files into it
+	tempDir = send_lib.prepare_temporary_directory()
```
Comment (xflord) [inline send/atlassian_mu:38]:
```suggestion
	with send_lib.prepare_temporary_directory() as tmpDir:
```
Please make sure to handle resources carefully

**Review by xflord — CHANGES_REQUESTED** [review]

**Review by mattjoke — APPROVED** [review]
:+1:

**Review by xflord — APPROVED** [review]
ok to merge if tested

**Comment by xflord** [comment]
While testing I fixed a couple bugs step by step to thoroughly test, ended up fixing the whole thing, whoops. https://github.com/CESNET/perun-services/pull/78

**Comment by xflord** [comment]
merged as part of https://github.com/CESNET/perun-services/pull/780

**Pull request closed without merging**